server/utils/bot/bot.go

```diff
@@ -54,8 +53,8 @@ func (bot *bot) Ensure(stored *model.Bot, iconPath string) error {
 		// Already done
 		return nil
 	}
-
-	botUserID, err := bot.pluginHelpers.EnsureBot(stored, plugin.ProfileImagePath(iconPath))
+	client := pluginapi.NewClient(bot.pluginAPI, nil) // driver passed as nil, as we don't need it
```

Comment:
> as we don't need it

Could this assumption change in the future? I'm actually not sure what the `driver` is used for.

Reply:
That driver is used for doing direct database queries, and the this created client is used just to do EnsureBot (which don't use direct). 

But still I'm 2/5 too... What's worse? to cross a dep to driver (which we don't need) or to bet on nil-driver hoping nobody uses it in the future...